include/model/Timestamp.hpp

```diff
@@ -0,0 +1,9 @@
+#include <chrono>
+
+namespace doma {
+namespace model {
+
+using Timestamp = std::chrono::system_clock::time_point;
```

Comment:
please include `ModelClock.hpp`, and use `ModelClock` type because we already defined it:
```c++
using Timestamp = ModelClock::time_point;
```